package.json

```diff
@@ -45,6 +45,8 @@
     "postpublish": "./.kokoro/publish-min.sh"
   },
   "dependencies": {
+    "@opentelemetry/api": "^1.7.0",
```

Comment:
Good catch! Done.

Reply:
Hi @gkevinzheng, 
It looks like our kokoro tests ran on NPM 6.14.18 which doesn't support auto-installment for peer-dependencies: https://github.blog/2020-10-13-presenting-v7-0-0-of-the-npm-cli/#

This is causing [system test module not found test failure](https://btx.cloud.google.com/invocations/1f915947-5739-426a-bc6f-36ce8878fb85/targets/cloud-devrel%2Fclient-libraries%2Fnodejs%2Fpresubmit%2Fgoogleapis%2Fnodejs-logging%2Fnode14%2Fsystem-test;config=default/log).

I have posted in JS-Team chat but it doesn't look like there is any solution so far. There is a plan to upgrade kokoro infra to Node18 but not sure when it will be done.

We will probably need to move this back to `Dependencies` instead of `peerDependencies` due to this limitation.

Reply:
Due to the limitation of low NPM version in kokoro system test infra, JS language team suggested us to import OpenTelemetry/api and re-export it for now to avoid test failure and guarantee compatibility : https://chat.google.com/room/AAAAW_Lho4o/ljx6XogZJMI
Once NPM version in Kokoro infra has upgraded, I will change this to peer dependency again.